src/apis/drive/samples/web/README.md

```diff
@@ -0,0 +1,36 @@
+# Drive v3 API Samples
+
+These samples allow you to list files on Google Drive using this library in the browser.
```

Comment:
are we assuming this is hand written or generated?

Reply:
Hand-written

Reply:
Yeah, I would really rather not :P